app/enums/JourneyType/JourneyType.scala

```diff
@@ -37,3 +39,19 @@ case object Manage extends Operation {
 case object Cease extends Operation {
   override val operationType = "CEASE"
 }
+
+case class Opt(optJourney: OptJourney) extends JourneyType {
```

Comment:
OptInOutJourney ? not sure, check the context

Reply:
The changes you've suggested make it clearer and will implement. `Opt(optJourney)` may be clearer as a result? When this class is called with the proposed changes, we would see `Opt(OptInJourney)` or `Opt(OptOutJourney)`

Reply:
I suspect you can leave it as is, its just a bit confusing when I try to read this separately from the actual usage;